lib/grpc/adapter/client.ex

```diff
@@ -0,0 +1,27 @@
+defmodule GRPC.ClientAdapter do
+  @moduledoc false
```

Comment:
Adding a moduledoc here is really important so that this module is discoverable in the docs